transformerlab/db.py

```diff
@@ -26,6 +26,15 @@ async def init():
     await db.execute(
         "CREATE TABLE IF NOT EXISTS dataset(id INTEGER PRIMARY KEY, dataset_id UNIQUE, location, description, size)"
     )
+
+    try:
+        await db.execute(
+            """ALTER TABLE dataset ADD COLUMN json_data TEXT DEFAULT '{}'"""
```

Comment:
Should be type JSON I think? See the create statement for model.

Reply:
Not sure if this has follow-on impact to anything but should probably do the same as model for consistency.